Sources/ParseSwift/Types/ParseError.swift

```diff
@@ -406,3 +406,90 @@ extension ParseError: CustomStringConvertible {
         debugDescription
     }
 }
+
+// MARK: Compare Errors
+extension Error {
+    /**
+     Validates if the given ParseError Code is equal to the error.
+     
+    **Example use case:**
+    ~~~
+    if error.equalsTo(.objectNotFound)  {
+        //Do stuff
+    }
+    ~~~
+     - parameter errorCode: The ParseError code you want to compare to
+     */
+    public func equalsTo(_ errorCode: ParseError.Code) -> Bool {
+        guard let error = self as? ParseError else {
+            return false
+        }
+        return error.code == errorCode
+    }
+
+    /**
+     Returns an optional ParseError if the given ParseError Code is equal to the error.
+     
+    **Example use case:**
+    ~~~
+    if let parseError = error.equalsTo(.objectNotFound)  {
+        print(parseError.description)
+    }
+    ~~~
+     - parameter errorCode: The ParseError code you want to compare to
+     
+     - returns: An optional ParseError
+     */
+    public func equalsTo(_ errorCode: ParseError.Code) -> ParseError? {
+        guard let error = self as? ParseError else {
+            return nil
+        }
+        if error.code == errorCode {
+            return error
+        } else {
+            return nil
+        }
+    }
+
+    /**
+     Validates if the given ParseError Codes contains the error.
+     
+    **Example use case:**
+    ~~~
+    if error.contains([.objectNotFound, .invalidQuery])  {
+        //Do stuff
+    }
+    ~~~
+     - parameter errorCode: The ParseError codes as **Array** you want to compare to
+     */
+    public func contains(_ errorCode: [ParseError.Code]) -> Bool {
```

Comment:
Since these will be an an array or variadic, `errorCode` should be plural, `errorCodes`. You should make the change on all methods that take an array or variadic input.